vassal-app/src/main/java/VASSAL/counters/Obscurable.java

```diff
@@ -368,6 +371,14 @@ protected void drawObscuredToOthers(Graphics g, int x, int y, Component obs, dou
           - .5 * zoom * obsBounds.height / 2),
         obs, zoom * 0.5);
       break;
+    case INSET2:
+      obscuredToMeView.draw(g, x, y, obs, zoom);
+      final Rectangle bounds2 = piece.getShape().getBounds();
+      final Rectangle obsBounds2 = obscuredToMeView.getShape().getBounds();
+      piece.draw(g, x - (int) (zoom * .8 * obsBounds2.width / 2  - .8 * zoom * bounds2.width/2),
+                    y - (int) (zoom * .8 * obsBounds2.height / 2 - .8 * zoom * bounds2.height/2),
```

Comment:
I I hear you can factor expressions like this:
```
0.4 * zoom * (obsBounds2.width - bounds2.width)
```?

Reply:
Haha I imagine. I just copied from some other piece of ass-crud without thinking about the math too much, but happy to clean it up.